docs/Core.rst

```diff
@@ -1,152 +1,197 @@
-they are re-created every time it is loaded.
+Multiple contexts can be specified by separating them with a
+pipe (``|``) - for example, ``@foo|bar|baz/foo`` would match
+anything under ``@foo``, ``@bar``, or ``@baz/foo``.
 
 Interactive commands like `liquids` cannot be used as hotkeys.
 
-Many commands come from plugins, which are stored in ``hack/plugins/``
-and must be compiled with the same version of DFHack.  Others come
-from scripts, which are stored in ``hack/scripts``.  Scripts are much
-more flexible about versions, and easier to distribute - so most third-party
-DFHack addons are scripts.
 
-Something doesn't work, help!
-=============================
-First, don't panic :)
+.. _kill-lua:
 
-Second, dfhack keeps a few log files in DF's folder (``stderr.log`` and
-``stdout.log``). Looking at these might help you solve the problem.
-If it doesn't, you can ask for help in the forum thread or on IRC.
+kill-lua
+--------
+Stops any currently-running Lua scripts. By default, scripts can
+only be interrupted every 256 instructions. Use ``kill-lua force``
+to interrupt the next instruction.
 
-If you found a bug, you can report it in `the Bay12 DFHack thread`_,
-`the issues tracker on github <https://github.com/DFHack/dfhack/issues>`_,
-or visit `the #dfhack IRC channel on freenode
-<https://webchat.freenode.net/?channels=dfhack>`_.
 
-=============
-Core Commands
-=============
-DFHack command syntax consists of a command name, followed by arguments separated
-by whitespace. To include whitespace in an argument, quote it in double quotes.
-To include a double quote character, use ``\"`` inside double quotes.
+.. _load:
+.. _unload:
+.. _reload:
 
-If the first non-whitespace character of a line is ``#``, the line is treated
-as a comment, i.e. a silent no-op command.
+load
+----
+``load``, ``unload``, and ``reload`` control whether a plugin is loaded
+into memory - note that plugins are loaded but disabled unless you do
+something.  Usage::
 
-When reading commands from dfhack.init or with the ``script`` command, if the final
-character on a line is a backslash then the next uncommented line is considered a
-continuation of that line, with the backslash deleted.  Commented lines are skipped,
-so it is possible to comment out parts of a command with the ``#`` character.
+    load|unload|reload PLUGIN|all
+
+Allows dealing with plugins individually by name, or all at once.
+
+
+.. _ls:
+
+ls
+--
+``ls`` does not list files like the Unix command, but rather
+available commands - first built in commands, then plugins,
+and scripts at the end.  Usage:
+
+:ls -a:         Also list scripts in subdirectories of ``hack/scripts/``,
+                which are generally not intended for direct use.
+:ls <plugin>:   List subcommands for the given plugin.
+
+
+.. _plug:
+
+plug
+----
+Lists available plugins (*not* commands implemented by plugins).
+
+``plug [PLUGIN|v]`` - list plugin state and detailed description.
```

Comment:
`|v` doesn't do anything currently, and I don't think it did in r3 either. This command does take multiple (optional) plugin names now, though.